kitsune/wiki/views.py

```diff
@@ -67,7 +67,12 @@
     send_contributor_notification,
     send_reviewed_notification,
 )
-from kitsune.wiki.utils import get_visible_document_or_404, get_visible_revision_or_404
+from kitsune.wiki.utils import (
+    FallbackCodes,
+    get_visible_document_or_404,
+    get_visible_document_or_fallback_or_404,
```

Comment:
maybe this visualization here helps to better understand what I am trying to say in my comment above.